plan/statistics/statistics.go

```diff
@@ -311,19 +378,191 @@ func estimateNDV(sc *variable.StatementContext, count int64, samples []types.Dat
 	return int64(estimatedDistinct), nil
 }
 
+// build4SortedColumn builds column statistics for sorted columns.
+func (t *Table) build4SortedColumn(sc *variable.StatementContext, offset int, result ast.RecordSet, bucketCount int64, isPK bool) error {
```

Comment:
The name for the third argument is not proper.  "result" is not a good name for input argument.